### PR TITLE
Updated Photorealistic Style Transfer

### PR DESCRIPTION
**Pull request by sagnik1511**
Contributed on the issue #60.

Please review @smaranjitghose  :)

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Please share a link to a Colab notebook with the same code for demonstration in the description of the PR

**Comment by sagnik1511** [comment]
here's the link  https://colab.research.google.com/drive/1pDIerDGoHO6Gl5547bvHrhYQeIpWPN0C?usp=sharing 

@smaranjitghose

**Comment by smaranjitghose** [comment]
> here's the link https://colab.research.google.com/drive/1pDIerDGoHO6Gl5547bvHrhYQeIpWPN0C?usp=sharing
> 
> @smaranjitghose

Read the paper once more and understand the difference between artistic style transfer and photo-realistic style transfer

[This is my previous notebook](https://nbviewer.jupyter.org/github/smaranjitghose/DeepHoli/blob/master/Deep_Holi.ipynb)

Use it as a reference.

Skip torch, use tf so that we can retrain using tfjs later

**Comment by sagnik1511** [comment]
Can you give one more day?? I can prepare it, and then calling for next PR.
and can I contribute this on the artistic style transfer?
Actually new to image processing

**Comment by sagnik1511** [comment]
updated the file with different images. I think it is okay now. @smaranjitghose 
Issue #60

File: Photorealistic Style Transfer/HRNET.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import torch
+import torch.nn as nn
+import torchvision
+import numpy as np
+
+IN_MOMENTUM = 0.1
+
```
Comment (smaranjitghose) [inline Photorealistic Style Transfer/HRNET.py:7]: single script for this please

**Comment by smaranjitghose** [comment]
Do a squash commit for the PR please

**Comment by smaranjitghose** [comment]
input

output


How is the photorealistic style transfer performed here😅.

Please take a week and understand the issue itself and if you wish give another PR

**Pull request closed without merging**